src_cpp/opennnBridge/choose_activation_function.h

```diff
@@ -0,0 +1,124 @@
+#pragma once 
+
+//#include <iostream>
+#include <vector>
+#include <string>
+#include "ModelParams.h"
+//#include "CustumNN.h"
+#include "definitionsNN.h"
+#include <map>
+
+
+//#include "Eigen/Core"
+//#include "unsupported/Eigen/CXX11/Tensor"
+//#include <eigen3/Eigen/Core>
+#include "../opennn/opennn/opennn.h"
+#include "nifppEigenExtensions.h"
+
+using namespace OpenNN;
+
+
+//
+
+
+void chooseActivationFunction(std::shared_ptr<OpenNN::NeuralNetwork> neural_network , nifpp::Tensor1D<Index> activations_functions)
+{
+   for(int i = 0; i < (int)((neural_network->get_trainable_layers_pointers()).size() ); i++){
+          
+         
+
+         //Pooling
+         if( (neural_network->get_trainable_layers_pointers()(i))->get_type_string() == "Pooling" )
```

Comment:
Comparing strings is inefficient (compare each letter to another).
Use get_type instead and compare it with enum number from Definition at line 61 of file layer.h (in opennn).
@evgenyan95

Reply:
don't work